extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-safeframe.js

```diff
@@ -649,5 +657,48 @@ describes.realWin('DoubleClick Fast Fetch - Safeframe', realWinConfig, env => {
         expect(resizeAmpAdAndSafeframeSpy).to.be.calledOnce;
       });
     });
+
+    /**
+     * Send a message requesting a shrink.
+     */
+    function sendShrinkMessage(height, width) {
+      const shrinkMessage = {};
+      shrinkMessage[MESSAGE_FIELDS.CHANNEL] = safeframeHost.channel;
+      shrinkMessage[MESSAGE_FIELDS.ENDPOINT_IDENTITY] = 1;
+      shrinkMessage[MESSAGE_FIELDS.SERVICE] = SERVICE.SHRINK_REQUEST;
+      shrinkMessage[MESSAGE_FIELDS.PAYLOAD] = JSON.stringify({
+        'uid': 0.623462509818004,
```

Comment:
Where does this uid value come from?

Reply:
Just sending a random number, which is what sf does. Although I guess this isn't really random as it's hardcoded, but it doesn't matter what it is .